outside-border-only

WildByDesign opened the pull request:
Add $iBorderStyle 9 = outside-border-only

WildByDesign committed:
Add files via upload

sven-seyfert commented:
Hi and thanks for you PR 👌 @WildByDesign .
Are you sure you want to have this empty header line (a empty line between headline and content)?

    ┌─────────────────────────────────────────────────────────────────────────────────────────┐
    │ Language     Popularity (%)     Job Demand                                  Typical Use │
    │                                                                                         │
    │ JavaScript             62.3      Very High            Web Development, Frontend/Backend │
    │ C#                     27.1        High         Game Development, Windows Apps, Web Dev │
    │ Go                     13.8       Growing            Cloud Services, System Programming │
    │ PowerShell             13.5   Low to Moderate     Task Automation, DevOps, System Admin │
    │ AutoIt                  0.5         Low               Windows GUI Automation, Scripting │
    └─────────────────────────────────────────────────────────────────────────────────────────┘

Or do you want this?

    ┌─────────────────────────────────────────────────────────────────────────────────────────┐
    │ Language     Popularity (%)     Job Demand                                  Typical Use │
    │ JavaScript             62.3      Very High            Web Development, Frontend/Backend │
    │ C#                     27.1        High         Game Development, Windows Apps, Web Dev │
    │ Go                     13.8       Growing            Cloud Services, System Programming │
    │ PowerShell             13.5   Low to Moderate     Task Automation, DevOps, System Admin │
    │ AutoIt                  0.5         Low               Windows GUI Automation, Scripting │
    └─────────────────────────────────────────────────────────────────────────────────────────┘

Best regards
Sven

sven-seyfert commented on lib/data-to-table.au3:
```diff
@@ -141,34 +142,34 @@ Func __SetConstants()
+            ['HeaderVerticalLeft',  '',    '',    '│',   '│',   '│',   '│',   '║',   '║',   '│'], _
+            ['HeaderSeparator',     '',    '│',   '│',   '│',   '│',   '│',   '║',   '│',   ' '], _
+            ['HeaderVerticalRight', '',    '',    '│',   '│',   '│',   '│',   '║',   '║',   '│'], _
+            ['HeaderLeft',          '',    '',    '├',   '│',   '├',   '├',   '╠',   '╟',   '│'], _
```
@WildByDesign
Why do you want to change HeaderLeft and HeaderRight for the default border style 4?
I guess it was a mistake, right?

sven-seyfert replied:
I will redo this change in one of my next commits. But no showstopper regarding merging this.

WildByDesign commented:
> Or do you want this?

You're right, this second option is what I was trying to achieve. Sorry about that. I guess I must not have understood all of it. But my goal was the second example with no header.

WildByDesign committed:
Add two border style options

9 = rounded-corner-outside-border-only-no-header
10 = double-outside-border-only-no-header

WildByDesign commented:
It wasn't until after my second morning coffee that I realized that the `x` in the `HasHeaderBorder` option needed to be removed.

I've added the following:
```
9 = rounded-corner-outside-border-only-no-header
10 = double-outside-border-only-no-header
```

Please let me know if there are any issues. Thank you.

sven-seyfert commented:
Thanks @WildByDesign. The commits are fine. I will merge this, add README adjustments and then I create release version v0.4.0 🤝 .
